association-backend/src/association_membership/tests/domain/test_entities.py

```diff
@@ -56,8 +56,5 @@ async def _():
         invoice_pdf="https://pdfpdf",
     )
 
-    assert len(subscription._add_stripe_subscription_payment) > 0
-    assert (
-        subscription._add_stripe_subscription_payment[0].stripe_subscription_id
-        == "cs_xxx"
-    )
+    assert len(subscription._add_payments) > 0
```

Comment:
wouldn't be better to test that the len is 1? 😊

Reply:
sure 👍